ui/prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "singleQuote": false,
+  "trailingComma": "es5",
+  "jsxBracketSameLine": false,
+  "tabWidth": 2,
+  "semi": true,
+  "printWidth": 80
+}
```

Comment:
missing empty line. maybe you can make it a rule? empty line have to be the last for all files

Reply:
The problem is that this is a json file and when saving, the last line is moved to the line with the last character. In the last committee after adding endOfLine": "auto" it is term 9, do I need to add empty line for json exactly ?